<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>617</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe Print</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="EncryptBtn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>280</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ROG Fonts</family>
     </font>
    </property>
    <property name="text">
     <string>Encrypt File</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>691</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ROG Fonts</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>File Encryption --- Caesar Cipher</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="DecryptBtn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>280</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ROG Fonts</family>
     </font>
    </property>
    <property name="text">
     <string>Decrypt File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="locateBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ROG Fonts</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Locate a File</string>
    </property>
   </widget>
   <widget class="QLabel" name="footerLbl">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>460</y>
      <width>541</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>All rights Reserved  Ibrahim Jaber 2020</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
